index.bs

```diff
@@ -543,18 +543,21 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 				and suffix (after stripping out `"web "`) passes the [=parsing a MIME type=] check.
 
 	<h3 id="unsanitized-data-types-x"><dfn>Unsanitized data types</dfn></h3>
+		<em>This section is non-normative.</em>
 
-			These data types MUST NOT be sanitized by UAs:
+			These data types MAY NOT be sanitized by UAs:
 
 			* image/png
 
 			* [=optional unsanitized data types=]
 
-			<dfn>Optional unsanitized data types</dfn> are [=representation/mime type=]s specified by the web authors that MUST NOT be sanitized by the user agent.
+			<dfn>Optional unsanitized data types</dfn> are [=representation/mime type=]s specified by the web authors that MAY NOT be sanitized by the user agent.
 			The valid [=optional unsanitized data types=] are listed below:
 
 			* text/html
 
+			<p class=note> [=optional unsanitized data types=] may not be supported by the user agent. In some privacy modes the [=representation/mime type=]s listed in [=optional unsanitized data types=] may not be unsanitized.</p>
```

Comment:
Which behavior has this behavior?

Reply:
@EdgarChen mentioned that in Firefox when privacy mode is on, the HTML content may not be unsanitized.

Reply:
Since this is Firefox specific behavior, we should check whether they want this added as non-normative text in the spec. If not, we should exclude it. Hopefully someone from Firefox can weigh in on this.

cc: @EdgarChen @saschanaz @smaug----

Reply:
Webkit won't support it, either, right? I think we could make it more generic, perhaps something like"UAs might not support this due to their privacy requirements" or such?

Reply:
That sounds good to me, thanks!